package.json

```diff
@@ -21,16 +21,19 @@
     "url": "https://github.com/chanzuckerberg/edu-design-system/issues"
   },
   "scripts": {
-    "build": "yarn build:clean && yarn build:tokens && yarn build:js && yarn build:styles",
+    "build": "yarn build:clean && yarn build:tokens && yarn build:js && yarn build:styles && yarn build:icons && yarn copy-icons-to-lib",
     "build:clean": "rm -rf lib/",
+    "build:icons": "svg-sprite-generate -d src/icons/ -o src/icons/spritemap/spritemap.svg && yarn generate-icon-types",
     "build:js": "tsc --project tsconfig.build.json",
     "build:storybook": "build-storybook -o storybook-static",
     "build:styles": "postcss \"src/**/*.css\" --dir lib/ --base src/ --verbose",
     "build:tokens": "node ./style-dictionary.config.js",
     "chromatic": "chromatic",
+    "copy-icons-to-lib": "copyfiles -u 1 src/icons/**/* lib",
```

Comment:
Part of build step in `next`, but am wondering if this is necessary if we just want devs using `<Icon>` and not the svgs/spritemap directly

Reply:
hmm yeah I think we need to export the spritemap b/c Icon file reads from it, but don't think we need all the raw svg files

Reply:
I'll export the spritemap only then